spec/factories.rb

```diff
@@ -1,30 +1,25 @@
-FactoryGirl.define do
-  factory :list_entry do
-    list
-    book
+FactoryGirl.define do  factory :list_entry do
```

Comment:
Unnecessary spacing detected.<br>Block body expression is on the same line as the block start.